schema.sql

```diff
@@ -8,4 +8,28 @@ id INT Generated always as Identity,
 	neutered BOOLEAN,
 	weight_kg Decimal,
 	primary KEY (id)
-);
\ No newline at end of file
+);
+ALTER TABLE animals
+ADD species VARCHAR(50);
+BEGIN TRANSACTION;
+UPDATE animals
+SET species= 'unspecified';
+ROLLBACK;
+BEGIN TRANSACTION;
+UPDATE animals
+SET species= 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals
+SET species= 'pokemon'
+WHERE species ISNULL;
```

Comment:
- [ ] IS NULL is written separately, kindly add a space after IS:
`UPDATE animals SET species = 'pokemon' WHERE species IS NULL;`